src/sw_models/SymmetrizedAligner.cc

```diff
@@ -38,6 +38,13 @@ LgProb SymmetrizedAligner::getBestAlignment(const vector<string>& srcSentence, c
 LgProb SymmetrizedAligner::getBestAlignment(const vector<WordIndex>& srcSentence, const vector<WordIndex>& trgSentence,
                                             WordAlignmentMatrix& bestWaMatrix)
 {
+  // if the source sentence or target sentence length is 0, return a blank result
+  if (srcSentence.size() == 0 || trgSentence.size() == 0)
+  {
+    bestWaMatrix.clear();
+    return 0.0;
```

Comment:
This method returns a log probability. You should use the `SMALL_LG_NUM` constant.

Reply:
Done.